srcStatic/GameModules/ConsoleModule.cpp

```diff
@@ -14,19 +14,13 @@ ConsoleModule::ConsoleModule(const std::string& moduleName) : DllModule(moduleNa
 			moduleDirectory + " : " + GetLastErrorString());
 	}
 
-
-	try {
```

Comment:
Remove this try/catch for module loading should also be removed from the ini module code.

Reply:
Ahh, good catch. Go for it.

Reply:
Just a reminder this suggestion seems to have been left out.

Reply:
I looked at this closer the other day and thought it wasn't warranted. Bad suggestion on my part.